Pipfile.lock

```diff
@@ -72,30 +64,6 @@
             ],
             "version": "==2.6.0"
         },
-        "backports.functools-lru-cache": {
```

Comment:
Now do you see why i want `--keep-outdated` to respect things that used to be in the pipfile even if we can't find a parent for them anymore?  This is a valid dependency!

Reply:
Indeed. But I'm curious why it is removed. I'm using python3 within the virtualenv and locking. Maybe you are using python2?

Reply:
I generate and update with both versions typically to make sure I don’t have failing builds, which is why I focused on the use case of not removing dependencies that the resolver thinks are invalid

Reply:
Understood, feels the same now